src/test/java/com/jcabi/xml/XMLDocumentTest.java

```diff
@@ -349,9 +349,10 @@ public void buildsDomNode() throws Exception {
      */
     @Test
     public void comparesToAnotherDocument() throws Exception {
+        final String xml = "<hi><dude>  </dude></hi>";
         MatcherAssert.assertThat(
-            new XMLDocument("<hi>\n<dude>  </dude></hi>"),
-            Matchers.equalTo(new XMLDocument("<hi><dude>  </dude></hi>"))
+            new XMLDocument(xml),
```

Comment:
There is no new line in this test, here is what you do:
``
 MatcherAssert.assertThat(new XMLDocument(xml), Matchers.equalTo(new XMLDocument(xml)));
``